### ISSUE TITLE
NOD | Confirmation page - You submitted the following information for the Board Appeal

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#81564]
## Value Statement

**_As a_** Veteran that has just submitted an appeal on VA.gov
**_I want to_** review save a copy of the information I submitted on the form
**_So that_** I can retain it for my records and reference it if I need to recall what I submitted for future filing

---

## Background Context
We want to provide a copy of submission back to Veterans after they submit. Although a PDF will be generated, Veterans should also be able to review their answers in plain HTML, and be able to save a copy either by printing, copying, or saving the page.

## Outcome, Success Measure, KPI(S), and Tracking Link
CTA button click for CURRENT print confirmation page event | Baseline: 89 clicks out of 564 submissions| Link to Data Source: TBD

## Acceptance Criteria
- [ ] Users are able to reach the confirmation page
- [ ] Users are able to review all of their data from the form, in plain language, and get a list of file names for the files they've uploaded (not the actual files themselves)
- [ ] If users do not provide an answer for an optional question, they will see it represented in the summary as 'Not selected'
- [ ] Users are able to print their data (data is queried to remove save PDF and print actions)
- [ ] Users are able to save their data
- [ ] Users are able to copy their data (as text onto another document)
- [ ] Screen reader users are able to navigate to sections of responses using headers
- [ ] Screen reader users are able to identify the questions and answers (paired together)

## Designs and Build Notes
- This ticket is for the HTML that appears on the bottom of the confirmation page displaying the info submitted by the Veteran
- [FIGMA](https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=831-76228&mode=dev)

### Enablement team (if needed)
@ member of team

## Out of scope
- Content updates that need approval
- PDF link
- Confirmation email
- .txt download
- Supplemental claim and HLR confirmation pages
- Access to uploaded evidence files

## Tasks
- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready
- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by eileen-coforma** [comment]
Asking Julie for use case:
For fields that are optional and not selected, what should be the default view for users? Is it OK to say "no" or should we say "not selected", since there wasn't a specific action on the user's side.

**Comment by Mottie** [comment]
Form data added to confirmation page work completed by:
- https://github.com/department-of-veterans-affairs/vets-api/pull/16642 (Add feature flag)
- https://github.com/department-of-veterans-affairs/vets-website/pull/29636 (Add data to confirmation page)
- https://github.com/department-of-veterans-affairs/vets-website/pull/29704 (Update confirmation page unit tests)

**Comment by eileen-coforma** [comment]
From Julie: We should indicate that an option has not been selected.
[Asking CAIA](https://dsva.slack.com/archives/C01K37HRUAH/p1715885024926239) for standard language between Not selected , Answer not selected , Left blank, or something else?

**Comment by eileen-coforma** [comment]
Moving this to 'blocked' until we get a response from CAIA. I followed up with them today.

**Comment by eileen-coforma** [comment]
Language confirmed with CAIA: Not selected. Design has been updated with a note to updated behavior.

**Comment by eileen-coforma** [comment]
@Mottie This ticket is unblocked. AC updated with new behavior:

If users don't answer an optional question that's a checkbox on the form, they will see 'Not selected' in the confirmation review response.

**Comment by Mottie** [comment]
This change is now available for testing in staging.

**Comment by saderagsdale** [comment]
@eileen-coforma @Mottie Moving this back to Blocked until we have final content approval from Angie at BVA.
